servmanager/servmanager.go

```diff
@@ -308,20 +315,20 @@ func (srvMngr *ServiceManager) reloadService(srviceName string) (err error) {
 	if srv.ShouldRun() {
 		if srv.IsRunning() {
 			if err = srv.Reload(); err != nil {
-				utils.Logger.Err(fmt.Sprintf("<%s> Failed to reload <%s>", utils.ServiceManager, srv.ServiceName()))
+				utils.Logger.Err(fmt.Sprintf("<%s> Failed to reload <%s> because: %s", utils.ServiceManager, srv.ServiceName(), err))
```

Comment:
the error should look like: fmt.Sprintf("<%s> failed to reload <%s> err <%s>", utils.ServiceManager, srv.ServiceName(), err))